script/lib/patches.py

```diff
@@ -1,17 +1,20 @@
 import os
 import sys
 
-import git
+import yaml
```

Comment:
`yaml` can be imported only after the PATH is changed: `sys.path.append(PYYAML_LIB_DIR)`